### ISSUE TITLE
[Alpha][Bug] 

### ISSUE THREAD

**Issue by Acwildweb** [IceWhaleTech/CasaOS#2039]
### What happened?

I installed casaos on a machine where docker container was already present with some containers. By turning on the casaos panel I see all the containers previously installed at the bottom. If I click on them nothing happens other than a speech bubble appears saying import to casaos. I read several discussions where it was recommended to update to the latest version and in that case I should have seen three vertical dots next to each icon where I could make some choices and then import them into casaos. but in this version which is the last nothing happens. Can you help me solve it. Thank you

### Screenshots

![Screenshot](https://github.com/user-attachments/assets/505fb077-b7c1-4e68-ae43-f2c65bfd3baa)

**Comment by Gendmyb** [comment]
i found it,the app will be add after you click the docker app which have not be build.it will take a long time and the docker app will display no run even though it's running and status healthy！

**Comment by Acwildweb** [comment]
sorry but I didn't understand well! should I just click on one of the icons under legacy applications and wait? Because if I understood what you were saying correctly. It doesn't seem to work and everything remains as before.

**Comment by Gendmyb** [comment]
> sorry but I didn't understand well! should I just click on one of the icons under legacy applications and wait? Because if I understood what you were saying correctly. It doesn't seem to work and everything remains as before.

yes bro,I mean, there's an app that I haven't built yet, and I hover my mouse over the icon, and there's nothing else I can do to make a choice, just like the problem you described. Then I tried clicking on the icon, and there was no obvious response. When I went back in after a period of time, the application was already built and didn't exist in the unbuilt column. However, when I first tried to see if the newly built application was useful, I found that the icon didn't work properly: its parameters, such as the environment configuration, were correct, it's the same as the Docker compose file I wrote by hand, but when I hit start, the icon is always grayed out instead of lit up, but when I see it running in docker ps on the command line interface, the docker app, however, was successfully launched.

**Comment by Acwildweb** [comment]
> > sorry but I didn't understand well! should I just click on one of the icons under legacy applications and wait? Because if I understood what you were saying correctly. It doesn't seem to work and everything remains as before.
> 
> yes bro,I mean, there's an app that I haven't built yet, and I hover my mouse over the icon, and there's nothing else I can do to make a choice, just like the problem you described. Then I tried clicking on the icon, and there was no obvious response. When I went back in after a period of time, the application was already built and didn't exist in the unbuilt column. However, when I first tried to see if the newly built application was useful, I found that the icon didn't work properly: its parameters, such as the environment configuration, were correct, it's the same as the Docker compose file I wrote by hand, but when I hit start, the icon is always grayed out instead of lit up, but when I see it running in docker ps on the command line interface, the docker app, however, was successfully launched.

Ah ok now it's clearer to me. I thought you were giving me a solution to the problem but instead you only described your experience which unfortunately doesn't work in my installation having tried clicking several times and waiting for hours. I just hope that a developer is able to fix the problem or at least indicate when it will be done. Thanks anyway for your experience and support.

**Comment by Acwildweb** [comment]
I finally solved it by downgrading to an older version where everything seems to work better. It's absurd that with the advancement of the version, instead of improving the software, they break other things!

**Comment by StrayChild01** [comment]
It happened to me, but I didn't want to downgrade. In my case, I had to resort to the following:
- Check the names of the containers that are running (the ones shown as legacy) by running:
`docker container list`
- Check the volumes used by the all the containers by issuing the following command:
`docker ps -a --format '{{ .ID }}' | xargs -I {} docker inspect -f '{{ .Name }}{{ printf "\n" }}{{ range .Mounts }}{{ printf "\n\t" }}{{ .Type }} {{ if eq .Type "bind" }}{{ .Source }}{{ end }}{{ .Name }} => {{ .Destination }}{{ end }}{{ printf "\n" }}' {} | tee -a /tmp/my-container-volumes.txt` 
- Stop each container manually, for example the id 97e242ebb8bc:
`docker container stop 97e242ebb8bc` 
- Reinstall the application by using the volumes shown in /tmp/my-container-volumes.txt  by using the "custom install" button.
- Remove the container since we won't use it anymore:
`docker container rm 97e242ebb8bc` 

I know this is far from ideal and I remember that I didn't have to to that before, but it allowed me to do some cleaning because the old images not being used are still there using disk space.
